features-json/css-writing-mode.json

```diff
@@ -0,0 +1,232 @@
+{
+  "title":"CSS Writing Mode",
+  "description":"Property to define whether lines of text are laid out horizontally or vertically and the direction in which blocks progress.",
+  "spec":"https://drafts.csswg.org/css-writing-modes-3/#block-flow",
+  "status":"cr",
+  "links":[
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode",
+      "title":"MDN article"
+    }
+  ],
+  "bugs":[
+
+  ],
+  "categories":[
+    "CSS"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"a x #1",
+      "10":"a x #1",
+      "11":"a x #1"
+    },
+    "edge":{
+      "1":"a x #1"
```

Comment:
@ai @fyrd This is incorrect. Edge supports the standard syntax. Also the #1 note is a little vague. They were the standard syntax from SVG, but as often happens with CSS, they were renamed after IE implemented them: http://www.w3.org/TR/2003/CR-css3-text-20030514/#Progression